uiowa/uiowa
# Feature: page title template refactor

Resolves https://github.com/uiowa/uiowa/issues/2674. 

This pr removes all the duplicate page title templates that I could find and sets a page_title_alter function to set a custom page template for node titles.   

# How to test

`blt ds --site=sandbox.uiowa.edu (enable topic_collection, enable event split, enable areas of study split)`

Run `drush @sandbox.local cim` and test the article, page, topic collection, event, and areas of study content types.  Verify that the page title for each is using `field--node--title.html.twig` and that there are not empty/duplicate `<h1>` tags. 


`blt ds --site=grad.uiowa.edu`
- Test on custom grad content types mentor, thesis defense, scholar and verify that there are not empty/duplicate `<h1>` tags. 

`blt ds --site=admissions.uiowa.edu`
- Test on custom taxonomy pages and view pages like https://admissions.local.drupal.uiowa.edu/colleges/college-dentistry and https://admissions.local.drupal.uiowa.edu/areas-of-study and verify that the page title is using the `page-title.html.twig` template.   

`blt ds --site=hawkeyemarchingband.uiowa.edu`
- Test on v2 pages and verify that there are no issue with page titles. 

## User story

As a developer, I can create a new content type without having to add a field template file for the page title
